d2l-floating-buttons.html

```diff
@@ -111,9 +115,9 @@
 				window.addEventListener('scroll', this._reposition);
 
 				this._isRTL = (getComputedStyle(this._container).direction === 'rtl');
-				this._reposition();
 
 				Polymer.RenderStatus.afterNextRender(this, function() {
+					this._reposition();
```

Comment:
Does moving `_reposition` fix a bug?  I vaguely recall looking at this previously, and am worried it was left outside of the `afterNextRender` as a result of it introducing a bug.